cypress/e2e/tests/rbac/disable_keycloak.test.ts

```diff
@@ -47,7 +47,7 @@ describe(["@tier4"], "Perform certain operations after disabling Keycloak", func
     });
 
     it.skip("Bug MTA-1152: Auth disabled, Verify presence of Review application button", function () {
-        Application.validateReviewButton(this.rbacRules);
+        // Application.validateReviewButton(this.rbacRules);
```

Comment:
Because the method doesn't exist and it was giving an error. Since I don't know if someone else was working on that test and that was its ony line I didn't want to remove it. But I can do it if needed.

![image](https://github.com/konveyor/tackle-ui-tests/assets/117646518/9ee86404-9153-4419-88ae-36cae8f8e1b9)

Reply:
@ibragins This bug is closed . Can you check this TC in a separate PR and Alex can leave it as it is .